osclib/request_splitter.py

```diff
@@ -89,13 +89,10 @@ def ring_get(self, target_package):
                 return ring[len(self.api.crings)+1:]
         return None
 
-    def devel_project_get(self, request, target_project):
-        # Preserve logic from adi and note that not Leap development friendly.
-        source = request.find('./action/source')
-        devel = self.api.get_devel_project(source.get('project'), source.get('package'))
-        if devel is None and self.api.project.startswith('openSUSE:'):
-            devel = self.api.get_devel_project('openSUSE:Factory', target_project)
-        return devel
+    def devel_project_get(self, target_project):
+        if self.api.project.startswith('openSUSE:'):
+            return self.api.get_devel_project('openSUSE:Factory', target_project)
+        return None
```

Comment:
that is a bit too oversimplified I think :) it stil does make sense to check for devel in the target (leap for example has a handful packages with devel project set). The previous mistake was to check the source. Factory should be used as fallback.